tests/examples/specific-fallback/dependencies.yaml

```diff
@@ -0,0 +1,27 @@
+files:
+  all:
+    output: conda
+    conda_dir: output/actual
+    matrix:
+      cuda: ["11.5", "11.8"]
+    includes:
+      - cudatoolkit
+channels:
+  - rapidsai
+  - conda-forge
+dependencies:
+  cudatoolkit:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+          - matrix:
+              cuda: "11.6"
+            packages:
+              - cudatoolkit=11.6
+          - matrix:
```

Comment:
I'd like a test where the empty matrix is at the top, and is always preferred over the other values.